package.json

```diff
@@ -1,10 +1,10 @@
 {
     "name": "@iota/is-ui-components",
     "type": "module",
-    "version": "0.0.44",
+    "version": "0.0.45",
     "description": "Svelte UI components for IOTA Integration Services",
     "scripts": {
-        "dev": "svelte-kit dev --port ${PORT:-3000}",
+        "dev": "svelte-kit dev --port ${PORT:-3044}",
```

Comment:
Set default port to 3000 (as convention)

Reply:
no it will block since port already is used